utilities/virt.py

```
@@ -1939,7 +1944,36 @@ def migrate_vm_and_verify(
+    def _abort_migration_wait(_migration: VirtualMachineInstanceMigration) -> None:
+        for pod in utilities.infra.get_pod_by_name_prefix(
+            client=_migration.client, pod_prefix=VIRT_LAUNCHER, namespace=_migration.namespace, get_all=True
+        ):
+            # Get status/events for PODs in non-running or failed state
+            if pod.status not in {Pod.Status.RUNNING, Pod.Status.COMPLETED, Pod.Status.SUCCEEDED}:
+                pod_events = [
+                    event["raw_object"]["message"]
+                    for event in pod.events(timeout=TIMEOUT_5SEC, field_selector="type==Warning")
+                ]
+                LOGGER.error(
+                    f"POD Conditions:\n {pod.instance.status.conditions[0]}\nPOD Events:\n {', '.join(pod_events)}"
```

Ruff (0.15.13)

[warning] Logging statement uses f-string

(G004)